build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:$versions.gradle_plugin"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$versions.kotlin_gradle_plugin"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10"
```

Comment:
I think it is best to continue using the variable, and just updated the variable version in the dependencies.gradle file.